operations/mimir/config.libsonnet

```diff
@@ -224,6 +224,10 @@
     distributorLimitsConfig: {
       'distributor.ingestion-rate-limit': $._config.limits.ingestion_rate,
       'distributor.ingestion-burst-size': $._config.limits.ingestion_burst_size,
+
+      // Assume a minimum of 100 samples per request
+      'distributor.request-rate-limit': $._config.limits.ingestion_rate / 100,
```

Comment:
In this file we also have per-tenant class overrides (e.g. see `extra_small_user`). This limit should be overridden for each class. That being said, I suggest to keep it disabled by default, so no need to do it in this PR (on the contrary, we should remove these 2 lines in this PR), but something to keep in mind when rolling out at Grafana Labs for testing (then we'll upstream the change to enable it by default once we got enough confidence).